### ISSUE TITLE
FIX for: _on_message() takes 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by mr-n30** [CaliDog/certstream-python#46]
Hello,

pip install certstream generates the following error:
![image](https://user-images.githubusercontent.com/69602501/114325256-c4d62b80-9af4-11eb-992d-661928a1c08a.png)

To fix it I did the following steps **(You may have to use pip and pip3 just in case)**:
```bash
pip uninstall certstream
pip uninstall websocket-client
git clone https://github.com/CaliDog/certstream-python.git && cd certstream-python
```
Now edit the requirements.txt file and change **websocket-client>=0.56.0** to **websocket-client==0.56.0**

Now run: `pip install -r requirements.txt`

Everything should work now. If not try the following as well:
pip3 install -r requirements.txt
python3 setup.py install

Now you can just run the Python script with:
```bash
python3 certstream-python/certstream/cli.py
```

**Note**
I'm using **Python 3.8.5**

**Comment by HelloXiaoTai** [comment]
This method solved my problem, thank you!

**Comment by christophercutajar** [comment]
Seeing the same error with python 3.9.1.

```
[ERROR:websocket] 2021-04-28 11:57:39,876 - error from callback <bound method CertStreamClient._on_message of <certstream.core.CertStreamClient object at 0x7fc84b42c640>>: _on_message() takes 2 positional arguments but 3 were given
```

**Comment by Fitblip** [comment]
Hi folks, this should be fixed with #42

**Comment by christophercutajar** [comment]
Confirmed that it's working now
